docs/conf.py

```diff
@@ -28,13 +28,12 @@
 
 sys.path.insert(0, os.path.abspath("."))
 sys.path.append(os.path.abspath("./_ext"))
+sys.path.append(os.path.abspath(".."))
```

Comment:
I wonder if we should choose which level we want to use to refer to modules outside of the main package with sphinx. It feels a bit overkill to add three nested levels to the path.

Reply:
I'll merge this for now to get the tests working then see about simplifying this.